src/www/system_certmanager.php

```diff
@@ -906,6 +950,7 @@ function internalca_change() {
                   <td style="width:22%"><a id="help_for_csr_sign_csr" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("CSR file");?></td>
                   <td style="width:78%">
                     <textarea name="csr" id="csr" cols="65" rows="7"><?=$pconfig['csr'];?></textarea>
+                    <a href="#" class="csr_info_for_sign_csr">Display CSR Content (TODO: better style button)</a>
```

Comment:
Try to add classes "btn btn-primary"

Reply:
I added, and changed to btn-secondary to make difference from important button. Thanks.